repos.yaml

```diff
@@ -678,3 +678,10 @@ repositories:
     maintainers:
       - email: shih5@chinaunicom.cn
         name: Hao Shi
+  - name: yugabyte
```

Comment:
This repository is already listed. A maintainer is already listed. Does the maintainers list for it need to be updated instead? You can find that at https://github.com/helm/hub/blob/e463279c7c3d334673a96f8126da62d922550c21/repos.yaml#L533-L537

Reply:
@mattfarina my bad, didn't realize Andrew already had submitted this request, I update this PR to reflect the maintainers.